storage/engine/engine.go

```diff
@@ -37,14 +37,21 @@ type Iterator interface {
 	// Seek advances the iterator to the first key in the engine which
 	// is >= the provided key.
 	Seek(key []byte)
+	// SeekReverse advances the iterator to the first key in the engine which
+	// is <= the provided key.
+	SeekReverse(key []byte)
 	// Valid returns true if the iterator is currently valid. An
 	// iterator which hasn't been seeked or has gone past the end of the
 	// key range is invalid.
 	Valid() bool
-	// Advances the iterator to the next key/value in the
+	// Next Advances the iterator to the next key/value in the
 	// iteration. After this call, the Valid() will be true if the
 	// iterator was not positioned at the last key.
 	Next()
+	// Prev make  the iterator backward to the previous key/value
+	// in the iteration. After this call, the Valid() will be true if the
```

Comment:
`s/the Valid()/Valid()/g`

Should also fix the Next comment which has the same grammatical error.